src/templates/visit.js

```diff
@@ -63,44 +63,64 @@ export default function VisitTemplate({ data, ...rest }) {
             <Prose>
               <Heading
                 level={1}
-                size="XL"
+                size='XL'
                 css={{
-                  fontWeight: '600',
+                  fontWeight: '600'
                 }}
                 data-page-heading
               >
                 <span className='visually-hidden'>{title}</span>
-                <span aria-hidden="true">{field_title_context}</span>
+                <span aria-hidden='true'>{fieldTitleContext}</span>
               </Heading>
-              <HTMLList data={field_visit} />
+              {fieldVisit && (
+                <>
+                  <List type='bulleted'>
+                    {fieldVisit.sort((a, b) => {
+                      return a.weight - b.weight;
+                    }).map(({ description }, i) => {
+                      return (
+                        <li key={i + description.processed}>
+                          <Html html={description.processed} />
+                        </li>
+                      );
+                    })}
+                  </List>
+                </>
+              )}
 
-              {field_access && (
-                <React.Fragment>
-                  <Heading level={2} size="M">
+              {fieldAccess && (
+                <>
+                  <Heading level={2} size='M'>
                     Getting here
                   </Heading>
 
-                  <Html html={field_access.processed} />
-                </React.Fragment>
+                  <Html html={fieldAccess.processed} />
+                </>
               )}
 
-              {field_amenities?.length > 0 && (
-                <React.Fragment>
-                  <Heading level={2} size="M">
+              {fieldAmenities && (
```

Comment:
```suggestion
              {fieldAmenities?.length > 0 && (
```
The conditional rewrite only checks to see whether or not `fieldAmenities` exists, which can return `true` if it's just returning an empty array (`[]`). The causes the `Amenities` section to appear, even if there is nothing to list (see [Asia Library](http://localhost:8000/locations-and-hours/asia-library)). The above suggestion checks to see if it does exist (`?`), and if it has a `length` greater than `0`.

Reply:
Great catch, and thanks for providing the Asia Library example. I wonder if I should do this check on the `fieldVisit `and `fieldAccess `fields as well?

Reply:
Great catch on `fieldVisit` and `fieldAccess`. I updated the `fieldAccess` because we weren't looking for an array. More specifically, we're looking for `fieldAccess.processed`.